webapp/src/pages/ClassOverlap.tsx

```diff
@@ -162,16 +175,18 @@ const ClassOverlap = () => {
                         }}
                       />
                     }
-                    label="Scale By Class"
+                    label="Scale By Class Size"
```

Comment:
I know it's not new, but I'm not sure about the capitalization. That's not really a title. Our switches for "Exclude post-processing" and for "Reorder classes" are not capitalized. Do you agree with that?
```suggestion
                    label="Scale by class size"
```

Reply:
I'll make the change.

Reply:
Totally fine, as long as we're consistent. Only thing is we should change it in the docs as well. To be consistent with the Exploration Space docs it looks like we should write it as `Scale by class size`. I can do that either in a little bit or on Monday.

Reply:
☝️ Made those changes in the docs!

Reply:
Can someone take a quick look at some point (like, next week) and just confirm that the formatting looks decent as it is now? I wasn't sure if I should also put the headers for the descriptions of each option in `ticks`. @JosephMarinier @gabegma

Reply:
Specifically, should the names of each option in Plot Options be in code blocks, too? (The ones that are't currently.) I wasn't sure if you were saying that they should or that you like the headers below in code blocks, sorry! @JosephMarinier 

![Screen Shot 2022-12-05 at 10 44 36 AM](https://user-images.githubusercontent.com/16842928/205680048-18ff4790-b479-498d-a2f2-2c66ff2803d5.png)

Reply:
Ah! I didn't realize you left two un-code-block-ed. Since there are no other words next to them (like "off" or "on" later in the text) there is no ambiguity there. Also, if they were code-block-ed, there would be no words in bold, so it might look weird. All that to say I like it as is.

Reply:
Great, thank you!

Reply:
Oh no, I missed some capitalizations! I will fix those.

Also, wait...what do you mean by _two_?